.gitignore

```diff
@@ -29,6 +29,9 @@ build
 *.exe
 *.app
 
+# Swap files
```

Comment:
This file shouldn't be here. Learn how to use private, global gitignore.
This file describes your environment, not a project-specifc generated files.